src/pet.js

```diff
@@ -0,0 +1,83 @@
+
+const vPet = {
```

Comment:
What does vPet stand for? Can you think of a better way to name this that explains to reader exactly what information this object contains from a quick glance?